zenodo/modules/spam/views.py

```diff
@@ -146,6 +172,7 @@ def _expand_users_info(results):
             r.update({
                 "full_name": user.profile.full_name,
                 "username": user.profile.username,
+                "flag": _evaluate_user_domain(user.email)
```

Comment:
This can go in the previous `r.update(...)` since all users have an `.email` (but not a `user.profile`)